### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by cutipu** [cutipu/HASOKI#10]
🥇  :smiley: Should we upgrade this tool?

**Comment by iloveyumyumshrimpnoodles** [comment]
Yeah, please rewrite the tool as the code is currently in such a horrible state it hurts my eyes looking at it.

The code also hurts performance a lot, and makes the project bloated too
